$MirOS: ports/mailnews/pine/patches/patch-imap_src_ipopd_ipop3d_c,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- imap/src/ipopd/ipop3d.c.orig	Fri Sep 17 02:26:12 2004
+++ imap/src/ipopd/ipop3d.c	Wed May  4 23:28:07 2005
@@ -57,7 +57,7 @@ extern int errno;		/* just in case */
 
 /* Global storage */
 
-char *version = "2004.89";	/* server version */
+char *version = "2004.89L";	/* server version */
 short state = AUTHORIZATION;	/* server state */
 short critical = NIL;		/* non-zero if in critical code */
 MAILSTREAM *stream = NIL;	/* mailbox stream */
